components/raftstore-v2/src/raft/peer.rs

```diff
@@ -382,4 +374,25 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     pub fn proposals_mut(&mut self) -> &mut ProposalQueue<Vec<CmdResChannel>> {
         &mut self.proposals
     }
+
+    pub fn generate_read_delegate(&self) -> ReadDelegate {
+        let region = self.region().clone();
+        let region_id = region.get_id();
+        let peer_id = self.peer().get_id();
+        ReadDelegate {
```

Comment:
Better create a constructor for `ReadDelegate`.

Reply:
Done.